packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -497,28 +497,18 @@ describe('KeyringController', () => {
         describe('when there is no existing vault', () => {
           it('should create new vault, mnemonic and keychain', async () => {
             await withController(
-              { cacheEncryptionKey },
-              async ({ controller, initialState, encryptor }) => {
-                const cleanKeyringController = new KeyringController({
```

Comment:
This test should have no existing vault, as the describe block suggests. In this case that was accomplished by throwing away the controller created by the `withController` helper, and creating a new one from scratch. This was unnecessary.

We can instead skip the vault creation and test with the generated controller, making the test much simpler.